src/commands/Testing/setqp.ts

```diff
@@ -0,0 +1,31 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+
+import { BotCommand } from '../../lib/BotCommand';
+import { MAX_QP } from '../../lib/constants';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			usage: '<qp:int{0,2147483647}>',
```

Comment:
you put the gp limit on qp?

Reply:
Resolved them all but the gear one. We don't use that anywhere currently but +hax, no need to change it's implementation until we really need it.